pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md

```diff
@@ -335,6 +336,35 @@ with this one:
 
 Save and exit the file.
 
+Now, if you choosed `Yes` when prompted for enabling SSL connection, get the CA certificate from the OVHcloud Control Panel:
+
+
+![Download certificate](images/postgresql-tuto-01-connect-strapi-to-managed-postgresql16.png){.thumbnail}
+
+and save the generated file into the `config` folder, just beside the `database.js` file, and give it the name `ca-certificate.crt`
+Now open the `config/database.js`, and edit the `ssl` block as:
```

Comment:
Now open the `config/database.js` file and modify the `ssl` block as follows:

Reply:
Done